src/App.tsx

```diff
@@ -1,41 +1,62 @@
-import React from 'react';
+import { useState } from 'react';
 import './App.scss';
 import './styles/general.scss';
 import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 
-const App: React.FC = () => (
-  <div className="App">
-    <header className="App__header">
-      <label>
-        Select a user: &nbsp;
+const App: React.FC = () => {
+  const [userSelect, setUserSelect] = useState<number>(0);
+  const [selectedPostId, setSelectedPostId] = useState<number>(-1);
```

Comment:
it is not necessary to set primitive types for useState
```suggestion
  const [selectedPostId, setSelectedPostId] = useState(0);
```

Reply:
There is no `0` id)